tests/unit/action-toolbar.spec.ts

```diff
@@ -16,7 +16,11 @@ describe('ActionToolbar', () => {
   it('should instantiate action toolbar buttons with right classes', () => {
     const store = setupMockStore(
       // TO_REMOVE: add a ff for geo category until development is done
-      buildStateWithOnePipeline([], { featureFlags: { BASEMAP_AS_USUAL_DATASOURCE: 'enable' } }),
+      buildStateWithOnePipeline([], {
+        featureFlags: { BASEMAP_AS_USUAL_DATASOURCE: 'enable' },
```

Comment:
Not related to this PR as it seems to come from before but: should the feature flag be disabled in that describe block, and enabled only in the `describe('with feature flags', () => {` block?
Otherwise seems like we never test the case without the feature flag